docs/source/start.rst

```diff
@@ -227,6 +227,11 @@ terminal.
 That's it! You're all set up to run pyQuil locally. Your programs will make requests to these server endpoints to compile your Quil
 programs to native Quil, and to simulate those programs on the QVM.
 
+**NOTE**: We are transitioning from using an HTTP ``quilc`` server to an RPCQ one. In the near term,
+``-S`` will spawn an HTTP server at port 6000 and an RPCQ server (accessible via tcp://domain:port)
+at port 5555. The RPCQ port is configurable with the ``-p`` option on ``quilc`` and the HTTP port
```

Comment:
Unfortunately, this isn't true: `-S` will ignore `-p` entirely and not modify either port; `-R` will take whatever instruction `-p` provides.

Reply:
Alright -- we'll have to bring that back at some point soon. In the meantime, I'll just drop the mention.